devbook.xsl

```diff
@@ -568,6 +606,16 @@
       </footer>
       <script src="https://assets.gentoo.org/tyrian/jquery.min.js"></script>
       <script src="https://assets.gentoo.org/tyrian/bootstrap.min.js"></script>
+      <script src="https://unpkg.com/lunr/lunr.js"></script>
```

Comment:
Generally relying on third parties here sucks a lot. We should move it to assets instead.

Reply:
I agree with this decision. We want to copy any assets we're using on the sites to assets.gentoo.org. This is the www version of distfiles.gentoo.org, which is important in the event that a CDN or external site disappears, we still have functional features on our site(s).

Reply:
As long as the appropriate party adds this file where it needs to be that is fine with me.